scala3doc/src/dotty/dokka/tasty/SyntheticSupport.scala

```diff
@@ -40,8 +40,13 @@ trait SyntheticsSupport:
   def isSyntheticField(c: Symbol) =
     c.flags.is(Flags.CaseAccessor) || (c.flags.is(Flags.Module) && !c.flags.is(Flags.Given))
 
+  // There seems to be no straightforward way to check if position exists so we catch AssertionError defaulting to true
```

Comment:
We should introduce a `hack*` method that checks if a `Position` exists. The fact that a `Position` can not exist is a bug in the Reflect API, there's an issue for it.

Reply:
Done